src/js/pages/Activity/News.jsx

```diff
@@ -606,7 +614,7 @@ const ShowMoreItemsWrapper = styled('div')(({ theme }) => (`
 
 const SignInSmallOptionsWrapper = styled('div')`
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
```

Comment:
Hi Almee, please check on this and roll it back to row if the change to column isn't correct. Thank you!